plugins/anilist.py

```diff
@@ -0,0 +1,630 @@
+        post_con = await AS.post(URL, json={'query': query, 'variables': vars})
+        json_data = await post_con.json()
+        return json_data
+
+
+def post_to_tp(a_title, content):
+    """ Create a Telegram Post using HTML Content """
+    tp = Telegraph()
+    tp.create_account(short_name=a_title)
+    post = tp.create_page(
+        title=a_title,
+        html_content=content,
+        author_name="@PhycoNinja13b",
+        author_url="https://t.me/PhycoNinja13b"
+    )
+    return f"https://telegra.ph/{post['path']}"
+
+
+def make_it_rw(time_stamp, as_countdown=False):
+    """ Converting Time Stamp to Readable Format """
+    if as_countdown:
+        now = datetime.now()
+        air_time = datetime.fromtimestamp(time_stamp)
+        return str(humanize.naturaltime(now - air_time))
+    return str(humanize.naturaldate(datetime.fromtimestamp(time_stamp)))
+
+
+@userge.on_cmd("anime", about={
+    'header': "Anime Search",
+    'description': "Search for Anime using AniList API",
+    'flags': {
+        '-mid': "Search Anime using MAL ID",
+        '-wp': "Get webpage previews "},
+    'usage': "{tr}anime [flag] [anime name | ID]",
+    'examples': [
+        "{tr}anime 98444", "{tr}anime -mid 39576",
+        "{tr}anime Asterisk war"]})
+async def anim_arch(message: Message):
+    """ Search Anime Info """
+    query = message.filtered_input_str
+    if not query:
+        return
+    vars = {
+        'search': query,
+        'asHtml': True,
+        'type': "ANIME"
+    }
+    if query.isdigit():
+        vars = {
+            'id': int(query),
+            'asHtml': True,
+            'type': "ANIME"
+        }
+        if '-mid' in message.flags:
+            vars = {
+                'idMal': int(query),
+                'asHtml': True,
+                'type': "ANIME"
+            }
+
+    result = await return_json_senpai(anime_query, vars)
+    error = result.get('errors')
+    if error:
+        await CLOG.log(f"**ANILIST RETURNED FOLLOWING ERROR:**\n\n{error}")
+        error_sts = error[0].get('message')
+        await message.edit(f"**Error:** `[{error_sts}]`")
+        return
+
+    data = result.get('data')
+    data = data.get('Media')
+
+    # Data of all fields in returned json
+    id = data.get('id')
+    idmal = data.get('idMal')
+    romaji = data['title']['romaji']
+    english = data['title']['english']
+    native = data['title']['native']
+    format = data.get('format')
+    status = data.get('status')
+    synopsis = data.get('description')
+    season = data.get('season')
+    episodes = data.get('episodes')
+    duration = data.get('duration')
+    country = data.get('countryOfOrigin')
+    c_flag = flag.flag(country)
+    source = data.get('source')
+    coverImg = data.get('coverImage')['extraLarge']
+    bannerImg = data.get('bannerImage')
+    genres = data.get('genres')
+    genre = genres[0]
+    if len(genres) != 1:
+        genre = genres[0] + ", " + ", ".join(genres[1:])
```

Comment:
```suggestion
        genre = ", ".join(genres)
```
simply, this should work